File: adhoc/kleberbarros/1743.py
x = input().split()  
x = [int(i) for i in x]

y = input().split()  
y = [int(i) for i in y]

if(x[0] == y[0] or x[1] == y[1] or x[2] == y[2] or x[3] == y[3] or x[4] == y[4]):
    print('N')
else:
    print('Y')

File: adhoc/carolinacigerza/1743.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner(System.in);
        String a, b;
        int i;
        a = entrada.nextLine();
        b = entrada.nextLine();
        String[] itemsA = a.split(" ");
        String[] itemsB = b.split(" ");    
        for (i = 0; i < 5; i++) {
          if (Integer.parseInt(itemsA[i]) + Integer.parseInt(itemsB[i]) != 1) {
            System.out.println("N");
            return;
          }
        }
        System.out.println("Y");
        entrada.close();   
    }
}